src/app/dialogs/change-password-dialog/change-password-dialog.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="dialog-content-wrapper">
+  <div class="dialog-details">
+    <mat-icon class="icon" mat-dialog-close>arrow_back</mat-icon>
+    <h3 class="dialog-title">Password change</h3>
+  </div>
+
+  <mat-dialog-content>
+    <form [formGroup]="changePasswordForm" (ngSubmit)="onSubmit()">
+      <mat-form-field class="example-full-width">
+        <mat-label>old password</mat-label>
+        <input matInput type="password" formControlName="oldPassword">
+      </mat-form-field>
+
+      <mat-form-field class="example-full-width">
+        <mat-label>new password</mat-label>
+        <input matInput type="password" formControlName="newPassword">
+      </mat-form-field>
+
+      <mat-form-field class="example-full-width">
+        <mat-label>confirm password</mat-label>
+        <input matInput type="password" formControlName="confirmPassword">
+      </mat-form-field>
+    </form>
+  </mat-dialog-content>
+</div>
+
+<button class="dialog-btn btn" (click)="onSubmit()" [disabled]="!changePasswordForm.valid" mat-dialog-close>SAVE
+  CHANGES</button>
```

Comment:
```suggestion
<button class="dialog-btn btn" (click)="onSubmit()" [disabled]="!changePasswordForm.valid" mat-dialog-close>
    SAVE CHANGES
</button>
```

This formatting would be better